st3/sublime_lib/settings_listener.py

```diff
@@ -145,13 +145,29 @@ def on_setting_changed(
 
     The handler is not called every time the `settings` object changes,
     but only when the value derived using the `selector` argument changes.
-    If `selector` is callable, then the derived value is ``selector(self)``.
+    If `selector` is callable, then the derived value is ``selector(settings)``.
     If `selector` is a :class:`str`,
```

Comment:
I believe this to be the most common use case, so it should be explained first. Either way, a callable is probably last.

Reply:
Good point; I've moved the string case first.